src/app/shared/services/user.service.ts

```diff
@@ -23,15 +23,16 @@ export class UserService {
 
   public check(): void {
     const id = this.getUserId();
-
     if (id.length === 0) {
       this.store.dispatch(resetUser());
     }
 
     this.http.get(`users/${id}`).subscribe({
```

Comment:
it's a good practice, to write a returned type of response like for example
this.http.get<IUser>

than the is no need to use res as bla-bla

Reply:
done